### DISCUSSION TITLE
Games that use launcher

### DISCUSSION THREAD

**Discussion by FrippWidow** [m4dEngi/RemotePlayWhatever#14] [Q&A]
Howdy, thanks for the good work in this tool, i'm trying to play non-Steam Guilty Gear Revelator, it requires a launcher to proper work, i can't add the Game.exe on Steam cause the game just don't launch, but if i add the launcher, the game opens up but it's not on Steam anymore, i guess cause is a different process, so it doesn't have overlay and i can't use the tool. 
Any thoughts?

**Comment by m4dEngi** [comment]
Usually working steam overlay is a requirement, but for non-steam games RPW switches client to desktop streaming so technically it's possible to stream non-steam games without overlay. It's just not implemented in current builds.

Also desktop streaming mode is probably the cause of guest input bug that limits input options to keyboard & mouse.
